l1-contracts/scripts/verify.ts

```diff
@@ -19,6 +20,7 @@ function verifyPromise(address: string, constructorArguments?: Array<any>, libra
         address,
         constructorArguments,
         libraries,
+        // contract:"contracts/bridge/L1ERC20Bridge.sol:L1ERC20Bridge"
```

Comment:
same

Reply:
We need this to verify the bridge, so I will add a not here and leave it there